config/settings.yml

```diff
@@ -47,3 +47,26 @@ STANFORD_NETWORK:
   ranges: []
 BITLY:
   ACCESS_TOKEN: 'the_bitly_apikey'
+SEARCH_TIPS:
```

Comment:
Not super pleased w/ this, but there is a goal to make this somewhat easily editable (i.e. not buried in a class/view).

Happy to hear other possible approaches.

Reply:
What about putting in a separate config file and adding source at runtime (initializer?)

https://github.com/railsconfig/config#adding-sources-at-runtime

```ruby
Settings.add_source!("/config/catfacts.yml")
```

Reply:
Neat!  Didn't know about `add_source!`.  I'll give that a go.